react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -48,9 +48,12 @@
     private OnMediaLibraryButtonListener mOnMediaLibraryButtonListener;
     private OnReattachQueryListener mOnReattachQueryListener;
     private OnEditorMountListener mOnEditorMountListener;
+    private boolean mEditorIsMounted = false;
```

Comment:
It would be maybe cleaner to name it `isEditorMounted` as it's boolean.
and not sure if we need to set it as false, as Java does it automatically for us?

Reply:
good! 👌 addressed in 4f744ccc